.github/ISSUE TEMPLATES/config.yml

```
@@ -0,0 +1,13 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Contributing guidelines
+    url: https://github.com/cardano-base/blob/master/CONTRIBUTING.md
+    about: Some rules & processes we honor.
+
+  - name: Feature ideas
+    url: https://github.com/cardano-base/discussions/categories/ideas
+    about: Maybe someone else had the same or a similar idea already?
+
+  - name: All issues
+    url: https://github.com/cardano-base/issues
+    about: Check whether your issue is not already covered here.
```

File should end with a newline
```suggestion
    about: Check whether your issue is not already covered here.

```